src/SingleImagePage/imagesList.tsx

```diff
@@ -0,0 +1,41 @@
+export interface Api {
+    id: number;
+    sol: number;
+    cameraName: string;
+    cameraFullName: string;
+    imageUrl: string;
+    earthDate: string;
+    roverName: string;
+}
```

Comment:
I know you copied this code from some work in progress...
but this really isn't a great name

- its not an API - its a model of the result from one endpoint.
- its going to be really confusing when we have more that one thing we fetch from the API.

Could we change this to better reflect what it is? `RoverImage` or somthing?